### PR TITLE
Remove obsolete properties from doc files

### PR DESCRIPTION
**Pull request by severo**
See #2210.

### CODE REVIEW

File: docs/setup/postgresql.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ psql -d georchestra -f postgresql/080-geofence.sql
 
 ## OGC statistics schema
 
-If the **security proxy** is deployed and ```shared.ogc.statistics.activated``` is true in your setup (false by default):
+If the **security proxy** is deployed and ```OGC_STATISTICS``` is true in your setup ([true by default](https://github.com/georchestra/datadir/blob/18.06/analytics/js/GEOR_custom.js#L4)):
```
Comment (fvanderbiest) [inline docs/setup/postgresql.md]: rather: If SP's log4j uses the org.georchestra.ogcservstatistics.log4j.OGCServicesAppender as configured in https://github.com/georchestra/datadir/blob/docker-master/security-proxy/log4j/log4j.properties

This is more logical than refering to a client side property here (which is just a consequence).

File: ldap/README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ The basic users:
  * ```testeditor``` has the USER & GN_EDITOR roles. The password is **testeditor**.
  * ```testadmin``` has the USER, GN_ADMIN, ADMINISTRATOR and MOD_* roles. The password is **testadmin**
  * ```testdelegatedadmin``` has the USER role. Is able to grant the EXTRACTORAPP & GN_EDITOR roles to members of the psc & c2c orgs. The password is **testdelegatedadmin**
- * ```geoserver_privileged_user``` is a required user. It is internally used by the extractorapp, mapfishapp & geofence modules. The default password is ```gerlsSnFd6SmM``` (you should change it, and update the ```shared.privileged.geoserver.pass``` option in your shared.maven.filters file).
+ * ```geoserver_privileged_user``` is a required user. It is internally used by the extractorapp, mapfishapp & geofence modules. The default password is ```gerlsSnFd6SmM``` (you should change it, and update the datadir as explain in its [README](https://github.com/georchestra/datadir/blob/18.06/README.md)).
```
Comment (fvanderbiest) [inline ldap/README.md]: as explained

File: docs/setup/postgresql.md (hunk 1 above)
Comment (severo) [reply docs/setup/postgresql.md]: Fixed, thanks

File: ldap/README.md (hunk 2 above)
Comment (severo) [reply ldap/README.md]: Fixed, thanks

**Pull request merged**